src/app/app.js

```diff
@@ -4,26 +4,18 @@ import '../pages/home/home.js';
 import css from './app.css';
 
 class AppComponent extends LitElement {
-
   render() {
     return html`
       <style>
         ${css}
       </style>
       
       <div>
-
-        <section>
```

Comment:
hmm, I think using `<section>` would be considered good semantic practice right?  

In essence, my thought was that they would be considered analogous to rows or containers in Bootstrap, so a section for `<header>`, content, `<footer>`, etc.